crates/cxx-qt-gen/src/parser/cxxqtdata.rs

```diff
@@ -204,31 +213,38 @@ impl ParsedCxxQtData {
             if let ForeignItem::Fn(mut foreign_fn) = item {
                 // Test if the function is a signal
                 if attribute_take_path(&mut foreign_fn.attrs, &["qsignal"]).is_some() {
-                    let parsed_signal_method = ParsedSignal::parse(foreign_fn, safe_call)?;
+                    let parsed_signal_method = ParsedSignal::parse(foreign_fn.clone(), safe_call)?;
+
+                    let parsed_signal_method_self = ParsedSignal::parse(foreign_fn, safe_call)?;
+                    self.signals.push(parsed_signal_method_self);
 
                     self.with_qobject(&parsed_signal_method.qobject_ident)?
                         .signals
                         .push(parsed_signal_method);
-                // Test if the function is an inheritance method
-                //
-                // Note that we need to test for qsignal first as qsignals have their own inherit meaning
+                    // Test if the function is an inheritance method
```

Comment:
seems to be in lots of places :-) Is this jetbrains formatting subtly differently ? or has Rust 1.80 changed things :thinking: 

As it's creating noise unrelated to your change, either should be in it's own change if that is a cargo fmt change or we need to figure out why your IDE is subtly different :sweat_smile:

Reply:
Done